src/widgetastic_patternfly5/components/menus/select.py

```diff
@@ -182,3 +187,14 @@ def items(self):
 
 class CheckboxSelect(BaseCheckboxSelect, Dropdown):
     DEFAULT_LOCATOR = './/div[contains(@class, "-c-select")][1]'
+
+
+class BaseTypeaheadSelect(BaseSelect):
+    text = TextInput(locator=".//input")
+
+    def read(self):
+        return self.browser.get_attribute("value", self.text)
```

Comment:
Lets make this part of widget directly only single method.
Better to use `input` as name of child widget. 
```
input = TextInput(locator=".//input")
```